CHANGELOG

```diff
@@ -1,3 +1,128 @@
+Version 3.6 (2018-03-xx)
+  Friendica Core:
+    Updates to the translations (DE, EN_GB, EN_US, ES, FR, IT, ZH_CN) [translation teams]
+    Update for the Danish and French regions [Alkarex]
+    Update for the documentation [andyhee, annando, rabuzarus, ratten, rudlof, silke, tobiasd]
+    Updates to the themes [annando, fabrixxm, hoergen, rebeka-catalina, rabuzarus]
+    Enhancements to the ARIA support in frio [rabuzarus]
+    Enhancements to the DB handling and structure [annando]
+    Enhancements to the API [annando, fabrixxm, hypolite, rudlof]
+    Enhancements to the support of Open Graph with images [hoergen]
+    Enhancements to the Diaspora federation (participation signal, relay of dislikes, basic forum support for D*, Birthdays) [annando]
+    Enhancements to the OStatus federation [annando]
+    Enhancements to the handling of feed contacts [hypolite]
+    Enhancements to the display of threaded discussions (optional) [hypolite]
+    Enhancements to the display of events [hoergen]
+    Enhancements to the ACL dialog (selection of forums) [rabuzarus]
+    Enhancements to the handling of new connections [annando]
+    Enhancements to the vitality check of contacts [annando]
+    Enhancements to the deamon script [annando]
+    Enhancements to the federation stats [annando, tobiasd]
+    Enhancements to the interaction with puclic postings [annando]
```

Comment:
`public posts`

Shouldn't it be an addition?

Reply:
Viewing is interacting too ;-)

Reply:
Fair enough, the typo remains.